clearwater-radius-auth/usr/share/clearwater-infrastructure/scripts/clearwater-radius-auth

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Enable or disable RADIUS authentication based on whether there are any RADIUS
+# servers set up in /etc/pam_radius_auth.conf. The regex checks if there are any
+# lines where the first non-whitespace character is a # (i.e. a comment).
```

Comment:
The config file can contain three things:

* Lines that just include whitespace (includes blank lines).
* Comment lines (first non-whitespace character is a #)
* Server lines (first non-whitespace character is not a #).

The regex searches for whitespace followed by a non-whitespace non-# character, which means it contains a server line, which means RADIUS must be enabled. Does that answer your question?